communities.json

```diff
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "resourcefulbees.png",
```

Comment:
After merging, your project will be featured on our [Discord ❤  Open Source](https://discord.com/open-source) page. That page automatically applies a filter to your logo that makes it solid white except for the transparent portions.

The logo you are using makes use of color to distinguish different parts of it, can you confirm the overall shape is enough to distinguish your logo?

Reply:
Ive updated the logo to be more distinguishable.